pipeline.py

```diff
@@ -17,16 +17,17 @@
 import pathlib
 from typing import Any
 
+from aws_cdk import Stack
```

Comment:
I'd prefer to keep module namespace for the reader to quicker understand which module `Stack` belongs to:
```suggestion
import aws_cdk as cdk
```